calico-vpp-agent/cni/cni_server.go

```diff
@@ -319,6 +324,18 @@ func NewServer(v *vpplink.VppLink, rs *routing.Server, ps *policy.Server, prs *p
 	}
 	pb.RegisterCniDataplaneServer(server.grpcServer, server)
 	l.Infof("Server starting")
+
+	sc := statsclient.NewStatsClient("")
+	err = sc.Connect()
+	if err != nil {
+		server.log.WithError(err).Errorf("could not connect statsclient")
+		return nil, err
+	}
+	time.Sleep(time.Second * 15) //waiting for stats to get updated on vpp manager side
```

Comment:
We need a better way to wait for stats to be updated. The CNI server should not be started before vpp-manager has finished configuring VPP, are the stats not up-to-date at this point?